src/WebJobs.Script/Metrics/HostMetrics.cs

```diff
@@ -25,6 +25,8 @@ public class HostMetrics : IHostMetrics
         // FaaS Metrics
         public const string FaasInvokeDuration = "faas.invoke_duration";
 
+        public static readonly string FaasMeterVersion = typeof(HostMetrics).Assembly.GetName().Version?.ToString();
```

Comment:
So this version is going to change every build/release. What are the implications of that?

Reply:
It just lets listeners know who is publishing these meters. Mostly for diagnostics I believe. Changing version is not an issue with every release.